xarray/core/variable.py

```diff
@@ -28,12 +28,7 @@
 from .indexes import PandasIndex, wrap_pandas_index
 from .indexing import BasicIndexer, OuterIndexer, VectorizedIndexer, as_indexable
 from .options import _get_keep_attrs
-from .pycompat import (
-    cupy_array_type,
-    dask_array_type,
-    integer_types,
-    is_duck_dask_array,
-)
+from .pycompat import DuckArrayModule, integer_types, is_duck_dask_array
```

Comment:
```suggestion
from .pycompat import (
    cupy_array_type,
    dask_array_type,
    integer_types,
    is_duck_dask_array,
    sparse_array_type,
)
```

Reply:
This changes was causing test failures because it removed `DuckArrayModule` - I've fixed that in the latest commit.